ChangeLog.rst

```diff
@@ -1,3 +1,9 @@
+libfuse 3.17.1 (unreleased)
+===========================
+
+* Add statx for distributions with stax support
```

Comment:
Can you add that under 3.17? I created a branch, but didn't release it yet - will merge master into the 3.17 over the weekend?